packages/formation-react/src/components/OMBInfo/OMBInfo.jsx

```diff
@@ -108,12 +116,16 @@ OMBInfo.propTypes = {
   resBurden: PropTypes.number,
 
   /**
-   * OMB control number / form number
+   * OMB control number / form number. If this
+   * omitted, it will not show in the rendered component. Be sure to get
+   * approval from the design council before leaving it out.
    */
-  ombNumber: PropTypes.string.isRequired,
+  ombNumber: PropTypes.string,
 
   /**
-   * Form expiration date
+   * Form expiration date. If this
+   * omitted, it will not show in the rendered component. Be sure to get
+   * approval from the design council before leaving it out.
```

Comment:
Do you plan on making the `expDate` optional as well? It currently isn't.

Reply:
It was in the plans for the pull request but was pulled out due to a change in design plans.  I will remove the verbage